src/api-v3/paths/accounts/{address}/info.spec.ts

```diff
@@ -10,10 +10,6 @@ import accountInfoLedgerIndexFooResponseFixture from '../../../../fixtures/accou
 const path = '/v3/accounts/{address}/info';
 
 describe(path, () => {
```

Comment:
Is this change just piggy backing along or is this necessary?

Reply:
Just test cleanup piggy backing along